template-parts/acf-featured-pages.php

```diff
@@ -0,0 +1,68 @@
+<?php
+// check that the ACF plugin is activated
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+if ( is_plugin_active( 'advanced-custom-fields-pro/acf.php' ) ) {
+
+		if( have_rows('featured_pages') ):
+
+				echo '<hr class="c-divider">';
+				echo '<div class="o-layout  o-layout--wide">';
+
+				// loop through the rows of data
+				while ( have_rows('featured_pages') ) : the_row();
+
+						echo '<div class="o-layout__item  u-4/12@lg">';
+
+						// page header with title link
+						if ( have_rows('page_header') ) :
+
+								while ( have_rows('page_header') ) : the_row();
+										echo '<h3><a href="' . get_sub_field('page_link') . '">' . get_sub_field('page_title') . '</a></h3>';
+								endwhile;
+
+						endif;
+
+						// page body with any combination of videos, images, quotes, etc.
+						if ( have_rows('page_body') ) :
+
+								while ( have_rows('page_body') ) : the_row();
+
+										if( get_sub_field('video') ):
```

Comment:
This code is a repeat of the template part selector in content-secondary.php, I think we also use it in the primary content area. Time to extract out into it's own template part? Also, would be good to have the field selectors in the same order and use the same ones? Does it make sense to change this to a switch from nested ifs?

Reply:
Agreed. I've moved the logic into its own file and changed the ifs to a switch.